lib/content-services/src/lib/tag/services/tag.service.ts

```diff
@@ -105,6 +116,46 @@ export class TagService {
         return observableRemove;
     }
 
+    /**
+     * Creates tags.
+     *
+     * @param tags list of tags to create.
+     * @returns Created tags.
+     */
+    createTags(tags: TagBody[]): Observable<TagEntry[]> {
+        const observableAdd$: Observable<TagEntry[]> = from(this.tagsApi.createTags(tags));
+        observableAdd$.subscribe(
+            (tagsEntries: TagEntry[]) => this.refresh.emit(tagsEntries),
+            (err) => this.handleError(err)
+        );
+        return observableAdd$;
+    }
+
+    /**
+     * Find tags which name contains searched name.
+     *
+     * @param name Value for name which should be used during searching tags.
+     * @param skipCount Specify how many first results should be skipped. Default 0.
+     * @returns Found tags which name contains searched name.
+     */
+    searchTags(name: string, skipCount: number = 0): Observable<ResultSetPaging> {
+        const sortingByName: RequestSortDefinitionInner = new RequestSortDefinitionInner();
+        sortingByName.field = 'cm:name';
+        sortingByName.ascending = true;
+        sortingByName.type = RequestSortDefinitionInner.TypeEnum.FIELD;
+        return from(this.searchApi.search({
+            query: {
+                language: RequestQuery.LanguageEnum.Afts,
+                query: `PATH:"/cm:categoryRoot/cm:taggable/*" AND cm:name:"${name}*"`
+            },
+            paging: {
+                skipCount,
+                maxItems: 100
```

Comment:
Do we always want 100 entries to be fetched? Or should we maybe use `UserPreferencesService` as we do for pagination? @AleksanderSklorz What do you think?

Reply:
Let me test it how it behaves

Reply:
Works fine. Done. Thank you for suggestion :)

Reply:
Oh by the way we can also consider providing `maxItems` as optional method parameter, so then whoever wants to use it can decide if he wants to pass some value there, or leave it empty and then 100 is the default that API provides when `maxItems` is empty

Reply:
Sure I can add maxItems parameter to function. If anyone sets maxItems parameters then this value will be taken. In other case value from UserPreferencesService will be used.

Reply:
@MichalKinas done